src/drive/mobile/lib/reporter.js

```diff
@@ -39,7 +37,7 @@ const logMessage = (message, serverUrl, level = 'info', force) => {
       level
     })
     if (force) {
-      configureReporter(isEnabled)
+      configureReporter(false)
```

Comment:
For example here, you essentially disable the reporter after a message has been forcefully logged. But there's a good chance that the reporter was already active, and it should stay that way.